portfolio/src/main/webapp/script.js

```diff
@@ -66,7 +66,12 @@ function tryHarderAndGiveDaisy() {
 
 async function getHelloFromButton() {
     const response = await fetch('/data');
-    const text = await response.text();
-    const cutieTextContainer = document.getElementById('cutie-text');
-    cutieTextContainer.innerText = text;
+    const text = await response.json();
+    for (i = 0; i < text.length; i++){
```

Comment:
nit: If you don't explicitly need the index, you can do a `for of` loop

```javascript
for (const currentText of text) {

}
```

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...of